rep.go

```diff
@@ -55,9 +55,9 @@ func (rep *repSocket) Recv() (Msg, error) {
 	defer cancel()
 	var msg Msg
 	err := rep.sck.r.read(ctx, &msg)
-	if err == nil && len(msg.Frames) > 1 {
-		msg.Frames = msg.Frames[1:]
-	}
+	// if err == nil && len(msg.Frames) > 1 {
```

Comment:
just remove it instead of commenting it out.

Reply:
I left it in for easy verification of the fix. Will remove it in the final PR.